exchanges/newexchange.md

```diff
@@ -0,0 +1,1061 @@
+      "autoPairUpdates": true
+     },
+     "websocketAPI": true,
+     "websocketCapabilities": {}
+    },
+    "enabled": {
+     "autoPairUpdates": true,
+     "websocketAPI": false
+    }
+   },
+   "bankAccounts": [
+    {
+     "enabled": false,
+     "bankName": "",
+     "bankAddress": "",
+     "bankPostalCode": "",
+     "bankPostalCity": "",
+     "bankCountry": "",
+     "accountName": "",
+     "accountNumber": "",
+     "swiftCode": "",
+     "iban": "",
+     "supportedCurrencies": ""
+    }
+   ]
+  },
+```
+
+###### Available pairs will be automatically filled in the configs when wrapper functions are filled out and gocryptotrader is run with the new exchange enabled
+
++ 3) Add the currency structs in ftx_wrapper.go:
+
+###### Futures currency support:
+
+Similar to the configs, spot support is inbuilt but other asset types will need to be manually supported
+
+```go
+	spot := currency.PairStore{
+		RequestFormat: &currency.PairFormat{
+			Uppercase: true,
+			Delimiter: "/",
+		},
+		ConfigFormat: &currency.PairFormat{
+			Uppercase: true,
+			Delimiter: "/",
+		},
+	}
+	futures := currency.PairStore{
+		RequestFormat: &currency.PairFormat{
+			Uppercase: true,
+			Delimiter: "-",
+		},
+		ConfigFormat: &currency.PairFormat{
+			Uppercase: true,
+			Delimiter: "-",
+		},
+	}
+```
+
++ 4) Document the addition of the new exchange (FTX exchange is used as an example below):
+
+###### root Readme.md:
+```go
+| Exchange | REST API | Streaming API | FIX API |
+|----------|------|-----------|-----|
+| Alphapoint | Yes  | Yes        | NA  |
+| Binance| Yes  | Yes        | NA  |
+| Bitfinex | Yes  | Yes        | NA  |
+| Bitflyer | Yes  | No      | NA  |
+| Bithumb | Yes  | NA       | NA  |
+| BitMEX | Yes | Yes | NA |
+| Bitstamp | Yes  | Yes       | No  |
+| Bittrex | Yes | No | NA |
+| BTCMarkets | Yes | No       | NA  |
+| BTSE | Yes | Yes | NA |
+| COINUT | Yes | Yes | NA |
+| Exmo | Yes | NA | NA |
+| FTX | Yes | Yes | No |
+| CoinbasePro | Yes | Yes | No|
+| Coinbene | Yes | No | No |
+| GateIO | Yes | Yes | NA |
+| Gemini | Yes | Yes | No |
+| HitBTC | Yes | Yes | No |
+| Huobi.Pro | Yes | Yes | NA |
+| ItBit | Yes | NA | No |
+| Kraken | Yes | Yes | NA |
+| Lbank | Yes | No | NA |
+| LakeBTC | Yes | No | NA |
+| LocalBitcoins | Yes | NA | NA |
+| OKCoin International | Yes | Yes | No |
+| OKEX | Yes | Yes | No |
+| Poloniex | Yes | Yes | NA |
+| Yobit | Yes | NA | NA |
+| ZB.COM | Yes | Yes | NA |
+```
+
+###### exchanges\support.go:
+```go
+var Exchanges = []string{
+	"binance",
+	"bitfinex",
+	"bitflyer",
+	"bithumb",
+	"bitmex",
+	"bitstamp",
+	"bittrex",
+	"btc markets",
+	"btse",
+	"coinbasepro",
+	"coinbene",
+	"coinut",
+	"exmo",
+	"ftx",
+	"gateio",
+	"gemini",
+	"hitbtc",
+	"huobi",
+	"itbit",
+	"kraken",
+	"lakebtc",
+	"lbank",
+	"localbitcoins",
+	"okcoin international",
+	"okex",
+	"poloniex",
+	"yobit",
+    "zb",
+```
+
+###### exchanges\exchange_test.go:
+```go
+func TestExchange_Exchanges(t *testing.T) {
```

Comment:
Also need to increment the value in https://github.com/thrasher-corp/gocryptotrader/blob/master/gctscript/wrappers/gct/exchange/exchange_test.go#L57

Reply:
to 29? i changed it from 27 to 28 unless im misunderstanding

Reply:
As in add instructions to also increment the value in the gctsript exchange test too